scss/_buttons.scss

```diff
@@ -202,6 +204,20 @@
     text-decoration: $link-hover-decoration;
   }
 
+  // Boosted mod
+  &:focus:not(:hover) {
```

Comment:
I think there's a better way to handle this with `outline-color: var(--#{$variable-prefix}btn-active-border-color);` here instead of `box-shadow`. IMO, it's not an urge.